tests/32-action_options.sh

```diff
@@ -54,7 +54,7 @@ stest 0 mkdir -p "$TESTDIR"
 stest 0 touch "$TESTDIR/testfile"
 stest 0 singularity exec --home "$TESTDIR" "$CONTAINER" test -f "$TESTDIR/testfile"
 stest 0 singularity exec --home "$TESTDIR:/home" "$CONTAINER" test -f "/home/testfile"
-if [ -n "${SINGULARITY_OVERLAY_FS:-}" ]; then
+if [  "$SINGULARITY_OVERLAY_FS" = "1" ]; then
```

Comment:
Can you change this to be:
`if [  "x${SINGULARITY_OVERLAY_FS}" = "x1" ]; then`

Not _that_ important... but if `SINGULARITY_OVERLAY_FS` gets undefined you'll have a parse error with no left side.